src/VisualStudio/VisualStudioInteractiveComponents/AssemblyRedirects.cs

```diff
@@ -5,17 +5,17 @@
 
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.Scripting.dll")]
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.CSharp.Scripting.dll")]
-[assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
+// [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
```

Comment:
Chatting with @Pilchie, we're not sure if this is working right now. We either might be excluding or including VB in both the real product and dogfood...